.dumi/scripts/webmcp.js

```
@@ -0,0 +1,34 @@
+      },
+    },
+    function (input) {
+      return {
+        content: [
+          {
+            type: 'text',
+            text: `Search Ant Design docs at: https://ant.design/components/overview. Query: ${input.query}`,
+          },
+        ],
+      };
+    },
```

ESLint

[error] Unexpected function expression.

(prefer-arrow-callback)